Empire/Empire.cpp

```diff
@@ -2012,7 +2012,220 @@ void Empire::UpdateAvailableLanes() {
     m_pending_system_exit_lanes.clear(); // TODO: consider: not really necessary, & may be more efficient to not clear.
 }
 
-const std::map<int, float>& Empire::SystemSupplyRanges() const
+void Empire::UpdateSupply()
```

Comment:
you've done something wrong with the rebase. these changes are undoing recent commits to master.